data/bugzilla-queries/fixed_in_query.yaml

```diff
@@ -0,0 +1,16 @@
+- query:
+    product:
+        - Red Hat CloudForms Management Engine
+    include_fields:
+        - id
+        - summary
+        - component
+        - description
+        - status
+        - qa_contact
+        - creator
+        - assigned_to
+        - version
+        - fixed_in
+    fixed_in:
```

Comment:
So this yaml file actually just serves as a template. When the command is run, it pulls the current version of your appliance that's specified in `env.local.yaml` and modifies the query with it's version. Or if you pass the `--stream/--version` argument it will use that version.